src/libraries/System.Text.Json/gen/JsonSourceGenerator.Emitter.cs

```diff
@@ -1101,6 +1101,11 @@ private string GetRootJsonContextImplementation()
             {
                 string contextTypeRef = _currentContext.ContextTypeRef;
                 string contextTypeName = _currentContext.ContextType.Name;
+                int backTickIndex = contextTypeName.IndexOf('`');
```

Comment:
I'm curious, does this PR address https://github.com/dotnet/runtime/issues/61782 ?

Reply:
I don't think so because the https://github.com/dotnet/runtime/issues/61782 issue is about a serialization type.
Int his case the `contextTypeName` is used for context's constructor declaration. These changes are introduced to fix https://github.com/dotnet/runtime/issues/66880.